index.html

```diff
@@ -122,75 +129,97 @@ <h2>Multi-post Stories Gain+Glory</h2>
           <div class="my-works-text2">
             <h2>Languages</h2>
-            <ul class="built-with2">
-              <li>JavaScript</li>
-              <li>Ruby</li>
-              <li>Html</li>
-              <li>Css</li>
-            </ul>
+            <div class="my-work-wrapper">
+              <ul class="built-with2">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
           </div>
         </div>
         <div class="my-work-card2">
           <div class="card-image2"><img src="image/multiple-box.png" alt="eclipse icon"></div>
           <div class="my-works-text2">
             <h2>Frameworks</h2>
-            <ul class="built-with2">
-              <li>Bootstrap</li>
-              <li>Ruby on rails</li>
-              <li>RSpec</li>
-              <li>Capybara</li>
-              <li>Selenium</li>
-            </ul>
+            <div class="my-work-wrapper">
+              <ul class="built-with2">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
           </div>
         </div>
         <div class="my-work-card2">
           <div class="card-image2"><img src="image/lightning.png" alt="eclipse icon"></div>
           <div class="my-works-text2">
             <h2>Skills</h2>
-            <ul class="built-with2">
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepan</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
+            <div class="my-work-wrapper">
+              <ul class="built-with2">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepan</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </div>
           </div>
         </div>
       </div>
-      <div class="chat-text">
-         <h2>
-          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-         </h2>
-      </div>
     </section>
-    <footer>
-      <form action="https://formspree.io/f/xoqrojkw" method="POST">
-        <div class="input-field">
-          <input type="text" name="name" maxlength="30" class="text-input" placeholder="Full name" required>
-          <input type="email" name="Email" class="text-input" placeholder="Email address" required>
-          <textarea name="textarea" id="text-area" maxlength="500" placeholder="Enter text here" cols="30" rows="10" required></textarea>
-        </div>
-        <div class="button-wrapp">
-          <button type="submit" class="get-in-touch-button" id="contact-button">Get in touch</button>
+    <footer id="contact-me">
+      <div class="footer-wrapper">
```

Comment:
- [ ] As suggested by previous reviewer,please position the background image correctly in the footer section to match the Figma design and that will make your site looks beautiful .you got what it takes 👍 .


Figma design:

![image](https://user-images.githubusercontent.com/90524466/177749508-708d7024-ed5b-4946-9d20-9ac518e5153a.png)


Your design:
![image](https://user-images.githubusercontent.com/90524466/177749704-d901e04c-3fa2-4b0f-9c6d-b1ff4445e421.png)